pandas/io/excel/_openpyxl.py

```diff
@@ -22,53 +25,22 @@ def __init__(self, path, engine=None, mode="w", **engine_kwargs):
         if self.mode == "a":  # Load from existing workbook
             from openpyxl import load_workbook
 
-            book = load_workbook(self.path)
-            self.book = book
+            self.book = load_workbook(self.path)
         else:
             # Create workbook object with default optimized_write=True.
             self.book = Workbook()
 
             if self.book.worksheets:
-                try:
-                    self.book.remove(self.book.worksheets[0])
-                except AttributeError:
-
-                    # compat - for openpyxl <= 2.4
-                    self.book.remove_sheet(self.book.worksheets[0])
+                self.book.remove(self.book.worksheets[0])
 
     def save(self):
         """
         Save workbook to disk.
         """
-        return self.book.save(self.path)
-
-    @classmethod
-    def _convert_to_style(cls, style_dict):
```

Comment:
I don't see this method is used anywhere. And it doesn't affect any test. Also, it seems like openpyxl doesn't have a `style` module anymore.